pkg/trace/config/config.go

```diff
@@ -253,6 +252,29 @@ type DebuggerProxyConfig struct {
 	APIKey string
 }
 
+// AppSecConfig hold the AppSec reverse proxy settings.
+type AppSecConfig struct {
+	// Enabled reports whether AppSec is enabled.
+	Enabled bool
+	// MaxPayloadSize is the maximum HTTP body payload size the reverse proxy is allowed to copy.
```

Comment:
Can you please check the other structures in the file and make sure we are consistent on adding white space between one struct field definition and the next comment throughout? This structure has no spacing, while the below structure does.

Reply:
Fixed in https://github.com/DataDog/datadog-agent/pull/11324/commits/9718f7d1bd2bde255789cd2697bf1ce59549b57d